docs/optimization_guide/nncf/ptq/code/ptq_torch_fx.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#! [dataset]
+import nncf
+import torch
+
+calibration_loader = torch.utils.data.DataLoader(...)
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+calibration_dataset = nncf.Dataset(calibration_loader, transform_fn)
+#! [dataset]
+
+#! [quantization]
+import torchvision
+import openvino.torch
```

Comment:
Why did you add `import openvino.torch`?

Reply:
https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html#how-to-use
![image](https://github.com/user-attachments/assets/f4bb451a-61e6-4b67-a081-90c5709805d1)